partner/policy-management-service/pom.xml

```diff
@@ -123,7 +111,7 @@
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-auth-adapter</artifactId>
-					<version>${project.version}</version>
+					<version>1.2.0.1</version>
```

Comment:
Use the version 1.2.0.1-java21-SNAPSHOT version. Create a property for the version. Can't this be loaded from runtime?

Reply:
Used property as a version. This is profile specific openapi-doc-generate-profile . This is used to generate api json yaml file which is further used in api docs.